internal/pkg/admission/reconciler.go

```diff
@@ -111,25 +124,25 @@ func (r *Reconciler) ReconcileDeletion(
 
 func setFalseConditionType(
 	conditions *[]metav1.Condition,
-	conditionType policiesv1alpha2.PolicyConditionType,
+	conditionType string,
```

Comment:
Nitpicking, but why the change to string, and later the casting? Why not use the type from the get go?